guided-charts/aws-hyperpod/templates/traefik/service.yaml

```diff
@@ -29,11 +29,7 @@ spec:
     app: traefik
     component: router
   ports:
-  - port: 80
-    name: web
-    targetPort: web
-    protocol: TCP
   - port: 443
     name: websecure
-    targetPort: websecure
+    targetPort: web
```

Comment:
sanity-check: does this mean we are accepting unencrypted traffic on 443? If so, isn't it confusing?